packages/mobile/src/components/edit/PriceAndAudienceField/PremiumRadioField/TrackPriceField.tsx

```diff
@@ -1,19 +1,35 @@
-import { Text } from 'app/components/core'
+import { useField } from 'formik'
+
+import { Box, Flex } from '@audius/harmony-native'
+import { Switch, Text } from 'app/components/core'
 import { PriceField } from 'app/components/fields/PriceField'
 
 import { BoxedTextField } from './BoxedTextField'
 
 export const TRACK_PRICE = 'stream_conditions.usdc_purchase.price'
+export const IS_OWNED_BY_USER = 'is_owned_by_user'
 
 const messages = {
   title: 'Set a Price',
   description: 'The price to unlock this track (min $1)',
   label: 'Cost to Unlock',
   placeholder: '1.00',
-  usdc: '(USDC)'
+  usdc: '(USDC)',
+  publishingRights: {
+    checkboxLabel: 'Direct Publishing Payments',
+    confirmationText:
+      'In order to receive direct publishing payments from Audius, I hereby confirm:',
+    bulletPoints: [
+      'I own all publishing rights to this music, including performance rights',
+      'I am not registered with a Performing Rights Organization or collection society'
+    ]
+  }
 }
 
 export const TrackPriceField = () => {
+  const [{ value: isOwnedByUser }, _ignored, { setValue: setIsOwnedByUser }] =
```

Comment:
nit: can just use `_` instead of `_ignored`, think that's the pattern we use elsewhere

Reply:
eslint yells 😢

Reply:
oh sry, try just no string: `[{ value: isOwnedByUser }, , { setValue: setIsOwnedByUser }]`

Reply:
oh haha, yea was advised to move to the `_ignored` approach from [this comment](https://github.com/AudiusProject/audius-protocol/pull/10184#discussion_r1819604135)  from the web PR

Reply:
oic.. this is fine then